build.sbt

```diff
@@ -384,9 +385,19 @@ lazy val fontAwesome = project
   .dependsOn(common)
   .settings(
     name := "lucuma-react-font-awesome",
+    Compile / scalacOptions += "-language:implicitConversions",
     facadeSettings
   )
 
+lazy val fontAwesomeDemo = project
```

Comment:
We should add a `rootFontAwesome` that aggregate this demo project so that it gets compiled in CI.